### ISSUE TITLE
[a11y]: Progress Bar - Playground - JAWS is not announcing the error status

### ISSUE THREAD

**Issue by guidari** [carbon-design-system/carbon#16595]
### Package

@carbon/react

### Browser

Firefox

### Package version

v1.58.0

### React version

18.02

### Automated testing tool and ruleset

Manual QA

### Assistive technology

JAWS

### Description

From the Progress Bar - playground, set the status value to "error", go to the component and observe the jaws announcement.
JAWS is not announcing the error state of the progress bar. JAWS is announcing as "Loading" for the error state.

### Recommended Fix
Ensure JAWS should announce the state of the progress bar correctly

### Reproduction/example

https://react.carbondesignsystem.com/?path=/story/components-progressbar--playground&args=status:finished

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by heloiselui** [comment]
Since this playground example doesn’t exist in our Storybook, we’re not able to reproduce or verify the issue as described. I believe we can close this issue.
